src/core/services/compiler/compiler.spec.js

```diff
@@ -159,27 +160,135 @@ describe('$mdCompiler service', function() {
         expect(scope.myControllerAs).toBe(data.element.controller());
       }));
 
-      it('should work with bindToController', inject(function($rootScope) {
-        var called = false;
-        var data = compile({
-          template: 'hello',
-          controller: function($scope) {
+    });
+
+  });
+
+  [
```

Comment:
Add a test asserting that if `bindToController` is set to `false` the `$onInit` method is called anyway.